db/migrate/0_database_structure.rb

```diff
@@ -0,0 +1,449 @@
+# encoding: UTF-8
+class DatabaseStructure < ActiveRecord::Migration
+  def change
+    unless table_exists? :album_categories
+      create_table 'album_categories', force: :cascade do |t|
+        t.string 'name', limit: 255
+        t.text 'text', limit: 65535
+        t.boolean 'visible', limit: 1
+        t.datetime 'created_at'
+        t.datetime 'updated_at'
+      end
+    end
+    unless table_exists? :album_categories_albums
+      create_table 'album_categories_albums', id: false, force: :cascade do |t|
+        t.integer 'album_id', limit: 4
+        t.integer 'album_category_id', limit: 4
+      end
+    end
+    unless table_exists? :albums
+      create_table 'albums', force: :cascade do |t|
+        t.string 'title', limit: 255
+        t.text 'description', limit: 65535
+        t.string 'author', limit: 255
+        t.string 'location', limit: 255
+        t.boolean 'public', limit: 1
+        t.datetime 'start_date'
+        t.datetime 'end_date'
+        t.datetime 'created_at'
+        t.datetime 'updated_at'
+        t.string 'category', limit: 255
+        t.integer 'photo_category_id', limit: 4
+      end
+    end
+    unless table_exists? :album_images
+      create_table 'albums_images', id: false, force: :cascade do |t|
+        t.integer 'album_id', limit: 4
+        t.integer 'image_id', limit: 4
+      end
+    end
+    unless table_exists? :albums_subcategories
+      create_table 'albums_subcategories', id: false, force: :cascade do |t|
+        t.integer 'album_id', limit: 4
+        t.integer 'subcategory_id', limit: 4
+      end
+    end
+    unless table_exists? :cafe_works
+      create_table :cafe_works do |t|
+        t.datetime  :work_day
+        t.integer   :pass
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.